### PR TITLE
refactor: Small changes in flank-scripts module

### LINKED ISSUE #1431: Refactor flank-scripts to enable new feature implementation
`flank-scripts` modules need to be slightly refactored. Currently, It's difficult to add any new feature

### PR DESCRIPTION
**Pull request by pawelpasterz**
Fixes #1431 

No new logic implemented. Unit tests + integration tests should be enough.

### CODE REVIEW

**Comment by pawelpasterz** [comment]
@flank-it

**Review by adamfilipow92 — APPROVED** [review]
Looks good! 👍

**Comment by pawelpasterz** [comment]
> **Integration tests failed, you could see results [here](https://github.com/Flank/flank/actions/runs/450219441)**

IT failed due to quota limit exceeded 😢

**Comment by Sloox** [comment]
@pawelpasterz what exactly does this PR accomplish?

**Comment by pawelpasterz** [comment]
> @pawelpasterz what exactly does this PR accomplish?

This is, let's say, a 'refactor' part of my work on https://github.com/Flank/flank/issues/1422. I wanted to avoid PR with a huge scope so I divided my work into 2 PR (the second is not yet created as it depends on that one)

File: flank-scripts/src/main/kotlin/flank/scripts/zenhub/ZenHubAPI.kt [hunk 1]
```diff
@@ -10,9 +10,10 @@ import com.github.kittinunf.result.success
 import flank.scripts.utils.toJson
 import kotlinx.serialization.Serializable
 
-private const val FLANK_REPO_ID = 84221974
+const val FLANK_REPO_ID = 84221974
 internal const val ZENHUB_BASE_URL = "https://api.zenhub.com/p1/repositories/$FLANK_REPO_ID"
 
+// GET
```
Comment (piotradamczyk5) [inline flank-scripts/src/main/kotlin/flank/scripts/zenhub/ZenHubAPI.kt:16]: this is not GET query

Comment (piotradamczyk5) [reply flank-scripts/src/main/kotlin/flank/scripts/zenhub/ZenHubAPI.kt:16]:
```suggestion
```

**Review by piotradamczyk5 — APPROVED** [review]

**Pull request merged**